src/main/java/com/crowdin/client/core/http/impl/json/EmptyArrayToNullDeserializer.java

```diff
@@ -43,11 +42,11 @@ public Object deserialize(JsonParser p, DeserializationContext ctxt) throws IOEx
     }
 
     private static boolean isCollectionType(Class<?> type) {
-        return type.isArray() && Collection.class.isAssignableFrom(type);
+        return type.isArray() || Collection.class.isAssignableFrom(type);
```

Comment:
Please explain this change

Reply:
I think these conditions are mutually exclusive in the context of Java:
- If 'type.isArray()' returns' true', it means the given type is an 'array', but arrays in Java are not assignable from 'Collection.class', so 'Collection.class.isAssignableFrom(type') will always return false for 'array' types.
Conversely, if' Collection.class.isAssignableFrom(type)' returns 'true', it means the given type is a subclass of 'Collection', but such types are not arrays, so 'type.isArray()' will always return false.

Reply:
Thank you!